module/applications/actor/character-sheet-2.mjs

```diff
@@ -702,6 +702,14 @@ export default class ActorSheet5eCharacter2 extends ActorSheetV2Mixin(ActorSheet
         save, range, reference, toggle, suppressed, level
       } = data;
 
+      if ( foundry.utils.getType(save?.ability) === "Set" ) save = {
+        ...save, ability: save.ability.size > 2
+          ? game.i18n.localize("DND5E.AbbreviationDC")
+          : game.i18n.getListFormatter({ style: "narrow" }).format(
+            Array.from(save.ability).map(k => CONFIG.DND5E.abilities[k].abbreviation)
+          )
+      };
```

Comment:
Need the same treatment where these DCs appear on items in sheets. That's the 'roll' column for spells on character sheets, and I think NPC sheets also display this on items on the main tab as well as the spells tab. In those places it should probably use the "DC" label if the number of abilities is >1, I don't think there's room for 2 abilities in those columns.